src/sinks/nats.rs

```diff
@@ -56,17 +56,33 @@ pub struct NatsSinkConfig {
     )]
     pub acknowledgements: AcknowledgementsConfig,
 
-    /// A name assigned to the NATS connection.
+    /// A [name][nats_connection_name] assigned to the NATS connection.
```

Comment:
```suggestion
    /// A NATS [name][nats_connection_name] assigned to the NATS connection.
```

To be consistent with the other?